test/unit/smart_answer_flows/child_benefit_tax_calculator_flow_test.rb

```diff
@@ -152,19 +152,37 @@ class ChildBenefitTaxCalculatorFlowTest < ActiveSupport::TestCase
             assert_same @calculator, @new_state.calculator
           end
 
-          should "go to add_child_benefit_stop? question" do
-            assert_equal :add_child_benefit_stop?, @new_state.current_node_name
-            assert_node_exists :add_child_benefit_stop?
+          should "go to add_child_benefit_1_stop? question" do
+            assert_equal :add_child_benefit_1_stop?, @new_state.current_node_name
+            assert_node_exists :add_child_benefit_1_stop?
+          end
+        end
+      end
+
+      context "when answering a later child_benefit_x_start? question" do
+        context "when the date is valid" do
+          setup do
+            @calculator.tax_year = "2015"
+            setup_states_for_question(
+              :child_benefit_2_start?,
+              responding_with: "2015-06-09",
+              initial_state: { calculator: @calculator },
+            )
+          end
+
+          should "go to the corresponding add_child_benefit_x_stop? question" do
+            assert_equal :add_child_benefit_2_stop?, @new_state.current_node_name
+            assert_node_exists :add_child_benefit_2_stop?
           end
         end
       end
 
       # Q3c
-      context "when answering how_many_children_part_year? question" do
+      context "when answering child_benefit_1_stop? question" do
```

Comment:
Think this should be `"when answering add_child_benefit_1_stop? question"`

Reply:
Oops thanks for this and the one below.